setup.py

```diff
@@ -87,7 +87,7 @@
         'websocket-client',
         'yaspin',
         # KFP runtime dependencies
-        'kfp==1.6.3',
+        'kfp>=1.6.3<1.7',
```

Comment:
I'd prefer we use the `~` operator for this, which would be equivalent...
```suggestion
        'kfp~=1.6.3',
```